### PR TITLE
N36: Gamercade

### PR DESCRIPTION
**Pull request by RobDavenport**
Part of #1098

### CODE REVIEW

File: content/news/036/index.md [hunk 1]
```diff
@@ -90,6 +90,44 @@ include:
...
+
+The community around the project is small, but is looking to expand.
+Come on over to the [subreddit](Gamercade-Subreddit), or hang out and chat
+on [discord](Gamercade-Discord), where the developers interact with members
+and post updates daily. The project is newly [open source](Gamercade-Github) and 
+looking for contributors, suggestions, as well as awesome game demos.
+
+_Discussions: [Discord](Gamercade-Discord),
+[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/w8idew/announcing_gamercade_a_new_wasm_powered_fantasy/),
+[/r/fantasyconsoles](https://www.reddit.com/r/fantasyconsoles/comments/w8ics8/announcing_gamercade_a_new_wasm_powered_fantasy/)
```
Comment (AngelOnFira) [inline content/news/036/index.md:123]: This line needs a closing underscore for italics

```suggestion
_Discussions: [Discord](Gamercade-Discord),
[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/w8idew/announcing_gamercade_a_new_wasm_powered_fantasy/),
[/r/fantasyconsoles](https://www.reddit.com/r/fantasyconsoles/comments/w8ics8/announcing_gamercade_a_new_wasm_powered_fantasy/)_
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

**Comment by RobDavenport** [comment]
@AngelOnFira thanks for the find! Have made the fix as requested.

**Review by AngelOnFira — APPROVED** [review]
Thanks, LGTM :)

**Review by ozkriff — APPROVED** [review]
:+1:  (made a few small fmt/style tweaks for consistency)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**